### ISSUE TITLE
No edit/delete in flashcards?

### ISSUE THREAD

**Issue by Cosmopal** [srv-twry/My-Companion#4]
There is not option to edit or delete a flashcard or question once created.

**Comment by srv-twry** [comment]
@Cosmopal  Actually there is an option of deleting the flashcard or question , it's implemented as a swipe to delete feature. Even though that feature also needs improvement since the user won't realise that their is a swipe to delete feature. 
Editing feature is definitely a good idea and we must add it. I will edit this issue to focus on the editing part and create a separate issue to show animation or some change to make deletion obvious. :smile:

**Comment by Cosmopal** [comment]
Gitter channel @srv-twry ?

**Comment by srv-twry** [comment]
@Cosmopal No, As the number of contributors on this project is not so much.

**Comment by andyb129** [comment]
Hi, I've created a pull request for this.  Fairly new to the pull request system, so hopefully I've done it correctly?

**Comment by udhay24** [comment]
i will work on this

**Comment by eashl003** [comment]
Hi! I would love to work on this. Would I be able to get assigned to this?

**Comment by dr0pdb** [comment]
Hi @eashl003 , Sure go ahead.

**Comment by eashl003** [comment]
Thanks! @dr0pdb
